frontend/README.md

```diff
@@ -43,16 +43,14 @@ Project Root
 It is useful to have a live-updating dev-server during development, and most of the time you can use the production Explorer backend:
 
 ```
-$ npm run dev:production-wamp-testnet
```

Comment:
Hmm, and will it be testnet or mainnet? This raises more questions in my head than it solves.

Reply:
Oh, that's a good point, I changed links in `NEAR_NETWORKS` config in `frontend.env` so the network-changing menu will actually work.

Reply:
I believe it is actually more intuitive to change the network via interface.
I hope we'll be able to move backend to the unified system as well.

Reply:
> I believe it is actually more intuitive to change the network via interface.
> I hope we'll be able to move backend to the unified system as well.

From the scalability and security point of view we should not spend our time merging the backends together as it increases complexity and won't be used in production that way.

Reply:
It will get development process easier and help mimicry the production (where switching between the networks is one-click procedure).
It's not urgent at all, I'll mention you in the PR if I ever get to do that.